test/pdmtypes.jl

```diff
@@ -4,25 +4,23 @@ using Base.Test
 
 call_test_pdmat(p::AbstractPDMat,m::Matrix) = test_pdmat(p,m,cmat_eq=true,verbose=1)
 
-for T in [Float64,Float32]
-  #test that all external constructors are accessible
-  m = eye(T,2) ; @test PDMat(m,cholfact(m)).mat == PDMat(Symmetric(m)).mat == PDMat(m).mat == PDMat(cholfact(m)).mat
-  d = ones(T,2) ; @test PDiagMat(d,d).inv_diag == PDiagMat(d).inv_diag
-  x = one(T) ; @test ScalMat(2,x,x).inv_value == ScalMat(2,x).inv_value
-  #for Julia v0.3, do not test sparse matrices with Float32 (see issue #14076)
-  (T == Float64 || VERSION >= v"0.4.2") && (s = speye(T,2,2) ; @test PDSparseMat(s,cholfact(s)).mat == PDSparseMat(s).mat == PDSparseMat(cholfact(s)).mat)
```

Comment:
Either you're reading this backwards or I am - this was a test that only ran on new enough julia. So it should still run.